### ISSUE TITLE
Sound doesn't stop on returning to menu

### ISSUE THREAD

**Issue by furstenheim** [feross/webtorrent-desktop#599]
**What version of WebTorrent Desktop?** (See the 'About WebTorrent' menu)
Version 0.6.1 (0.92.0)
**What operating system and version?**
Linux
**What did you do?**
As I was playing a video I clicked on the back button to return to the main menu.
**What did you expect to happen?**
Return to menu and stop the sound.
**What actually happened?**
, it returned but the sound didn't stop. It's happened couple of times but I haven't been able to reproduce consciously. There are three erros on the console:

> undefined:1 Uncaught (in promise) DOMException: The play() request was interrupted by a call to pause().
> 0:1 GET http://localhost:36347/0 net::ERR_CONTENT_LENGTH_MISMATCH
> 0:1 GET http://localhost:36347/0 net::ERR_CONNECTION_REFUSED

**Comment by feross** [comment]
What Linux distro and version are you using?

**Comment by feross** [comment]
I can NOT reproduce this on Ubuntu 64-bit 16.04.

**Comment by furstenheim** [comment]
Ubuntu 64bit 14.04. It is something that almost never happens. It happened once during normal execution and then 4 more times trying to reproduce it. 
What I did in order to reproduce was repeatedly playing either of two videos (Sintel and Big Buck Bunny) and then going back to the menu. As I said most of the time everything work out fine. But these five times the sound kept on going and the three same lines were logged to the console.

**Comment by feross** [comment]
My guess is that is either related to this Chrome issue: https://bugs.chromium.org/p/chromium/issues/detail?id=593273

Or, it could be caused by an earlier exception. Let's close this for now, and re-open if we see it after the v0.7.0 release.

**Comment by feross** [comment]
I just witnessed this issue, so it's real. I was about to open a new issue, but remembered that this one already exists.

**What version of WebTorrent Desktop?** (See the 'About WebTorrent' menu)

v0.7.2

**What operating system and version?**

OS X 10.11.5

**What did you do?**

I watched @jhiesey click the AirPlay icon repeatedly as fast as he could.

**What did you expect to happen?**

The app should remain in a reasonable state.

**What actually happened?**

Multiple copies of the audio play at the same time, even though there are no video tags in the DOM. This makes no sense of course, but I did confirm that `document.querySelectorAll('video')` returned no elements.

This could be a Chrome bug. This exception shows up (https://bugs.chromium.org/p/chromium/issues/detail?id=593273) but that could be unrelated. If Chrome is indeed at fault, we need to produce a minimal test case to show them, because they're currently not treating that issue very seriously.
